home/build.gradle

```diff
@@ -1,16 +1,27 @@
 apply from: "$rootDir/gradle/feature.gradle"
+apply plugin: 'kotlin-android-extensions'
```

Comment:
what do you need this for? kotlin-android-extensions is deprecated, so might want to look for alternates https://android-developers.googleblog.com/2020/11/the-future-of-kotlin-android-extensions.html